src/cider/nrepl/middleware/test.clj

```diff
@@ -64,7 +64,8 @@
      (->> e
           .getStackTrace
           (map (partial st/analyze-frame namespaces))
-          (filter #(= (:class %) class-name))
+          (filter
+           #(clojure.string/starts-with? (:class %) (.getName (class f))))
```

Comment:
`(.getName (class f))` doesn't appear to be different from `class-name`, see surrounding when-let